config/routes.rb

```diff
@@ -40,6 +40,10 @@ def show_step(name)
     end
   end
 
+  resource :about, only: [:show] do
```

Comment:
This exposes an additional route `abouts#show`.
Also named routes become `privacy_about_path`.

I suggest the simpler form, because about controller is not resourceful:

`get 'about/privacy'`

This gets you a nicer named route `about_privacy_path` (and the same when we have other about pages).